go/app/main.go

```diff
@@ -55,8 +55,12 @@ func prepareDB() {
 	);
 	`)
 	statement.Exec()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
This err should be checked before Exec() is called.
Also, statement.Exec may return error, so it should be checked.
https://pkg.go.dev/database/sql#Stmt.Exec